MetaMask/metamask-mobile
# chore: xCode 16 beta build on bitrise

## **Description**

Updated the bitrise.yml file to use the newer version of Xcode 16 instead of 15.

## **Related issues**

Fixes: https://github.com/orgs/MetaMask/projects/60/views/9?pane=issue&itemId=71134639

## **Manual testing steps**

1. Go to `bitrise.yml`
2. Check line 1513 to see the Xcode version used by Bitrise in the build process.
3. Confirm the builds actually work in Bitrise

## **Screenshots/Recordings**

### **Before**

### **After**

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
